pkg/kubelet/kubelet.go

```diff
@@ -1841,7 +1887,22 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 		node.Status.Conditions = append(node.Status.Conditions, newCondition)
 		kl.recordNodeOnlineEvent()
 	}
+	// Get system OOM Information.
+	lastSystemOOM, timestamp, err := kl.getRecentSysOOMEvent()
```

Comment:
What is the policy we include a sys oom into our NodeCondition? When decide to exclude one?